internal/repositories/owned_vehicles_test.go

```diff
@@ -290,3 +291,100 @@ func (s *OwnedVehiclesRepoTestSuite) Test_GetOwnedVehicles_Pagination_NextPage()
 
 	s.Exactly(expected, res.Edges)
 }
+
+func Test_GetOwnedVehicles_Filters(t *testing.T) {
```

Comment:
We should have more tests, maybe testing the combinations mentioned by @elffjs 

- Filter by Owner (Done already)
- Filter by both owner and beneficiary
- Filter by invalid beneficiary, but valid owner
- Filter by invalid owner, but valid beneficiary
- Filter by invalid beneficiary and owner